### DISCUSSION TITLE
Windows App SDK 1.4.3 (1.4.231115000)

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/WindowsAppSDK#3974] [Announcements]
## Windows App SDK 1.4.3 (1.4.231115000)

Windows App SDK 1.4.3 is a stable servicing release that fixes critical bugs for 1.4. 

- If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app?tabs=desktop-csharp).

- If you are already using the Windows App SDK 1.4 and would like to update to the latest release, you can do so by following the [update instructions here](https://docs.microsoft.com/windows/apps/windows-app-sdk/update-existing-projects-to-the-latest-release).

Check out our [release notes](https://learn.microsoft.com/windows/apps/windows-app-sdk/stable-channel#version-143-14231115000) for the list of issues fixed in this release.

### Updating to this version

The 1.4.3 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.4.231115000

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.4.3'>Windows App SDK 1.4.3 (1.4.231115000)</a>.</em>

**Comment by JochemPalmsens** [comment]
With VS 17.8.3 and this version, our app still crashes on startup when the in-app toolbar is enabled (See [this](https://github.com/microsoft/microsoft-ui-xaml/issues/8806) issue. This is blocking us from moving from 1.3 to 1.4. Any idea when this will be fixed?

**Reply by duncanmacmichael** [reply]
Sorry, but I don't. @bpulliam do we have any updates on this?

**Reply by duncanmacmichael** [reply]
Update: We're looking into this and will update you as soon as we have new info, thanks for your patience.
